playwright/tests/item-page.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { test, expect } from "@playwright/test";
+import ItemPage from "../pages/item-page";
+
+let itemPage: ItemPage;
+
+// Probably a nicer way to do this than hardcoding but :shrug?
+const CAPTURE_UUID = "e36b2b90-740d-0136-0517-476ebf952da4";
+const ITEM_UUID = "7cd3acc0-5d7f-0134-12ab-00505686a51c";
```

Comment:
i don't mind the hardcoding at this phase tbh. if we decide to do some more serious mocking in these tests then we can definitely go that route, but for now seems fine.

my suggestion is to move these consts into the page file and make them part of the ItemPage class. something like:

```
static captureUuid: string = "e36b2b90-740d-0136-0517-476ebf952da4";
etc.
```